packages/berry-pnpify/package.json

```diff
@@ -4,7 +4,7 @@
   "sideEffects": false,
   "main": "./sources/index.ts",
   "bin": {
-    "pnpify": "./lib/bin.js"
+    "@berry/pnpify": "./lib/bin.js"
```

Comment:
This change looks strange to me. Is it intentional?

Reply:
Nope! Erroneous normalization 😶